layouts/shortcodes/observability_pipelines/processors/remap.md

```diff
@@ -10,7 +10,7 @@ To set up the add field processor:
 
 ##### Drop field
 
-Use **drop field** to drop a field from logging data that matches the filter you specify below.
+Use **drop field** to drop a field from logging data that matches the filter you specify below. The drop field can delete objects, so you can use it to drop nested keys.
```

Comment:
```suggestion
Use **drop field** to drop a field from logging data that matches the filter you specify below. It can delete objects, so you can use the processor to drop nested keys.
```